src/saleor_app/endpoints.py

```diff
@@ -38,46 +27,36 @@ async def install(
     _domain_is_valid=Depends(verify_saleor_domain),
     saleor_domain=Depends(saleor_domain_header),
 ):
-    target_url = request.url_for("handle-webhook")
-    domain = saleor_domain
-    auth_token = data.auth_token
-    try:
-        await install_app(
-            domain,
-            auth_token,
-            request.app.extra["saleor"]["webhook_handlers"].get_assigned_events(),
-            target_url,
-            request.app.extra["saleor"]["save_app_data"],
-        )
-    except (InstallAppError, GraphQLError):
-        raise HTTPException(
-            status_code=403, detail="Incorrect token or not enough permissions"
+    events = {}
+    if request.app.webhook_router.http_routes:
+        events[request.url_for("handle-webhook")] = list(
+            request.app.webhook_router.http_routes.keys()
         )
+    for event_type, sqs_handler in request.app.webhook_router.sqs_routes.items():
+        key = str(sqs_handler.target_url)
+        events.setdefault(key, [])
```

Comment:
Question: Wouldn't be easier to convert `events={}` into `events = defaultdict(list)` and skip setdefault to each event?